File: Division.java
public class Division extends Calculator{
    public Division () {
        operator = '/';
    }
    
    public double compute(double A, double B){
        if (B == 0) {
            System.err.println("Division by zero error");
            return 0;
        }
        return A / B;
    }
}

File: division.py
from flask import jsonify

def divide(number1, number2):
    try:
        result = number1 / number2
        response = {
            "status": 200,
            "result": result
        }
    except ZeroDivisionError:
        response = {
            "status": 400,
            "message": "division by zero error",
            "result": None
        }
    return jsonify(response)
